src/stories/Library/disclosure-controllable/DisclosureControllable.tsx

```diff
@@ -25,13 +25,22 @@ const DisclosureControllable: React.FunctionComponent<
     setIsOpen(!isOpen);
   }, [isOpen]);
 
+  const onKeyDown = useCallback(
+    (e: { key: string }) => {
+      if (e.key === "Enter" || e.key === " ") {
+        toggleOpen();
+      }
+    },
+    [toggleOpen]
+  );
+
   const disclosureId = `disclosure-${id}`;
   return (
     <div className={detailsClassName}>
       <div
         className={summaryClassName}
         onClick={toggleOpen}
-        onKeyDown={toggleOpen}
+        onKeyDown={onKeyDown}
```

Comment:
I am not sure how this is related to the change at hand.

The mention in the commit comment does not help me much. The change might be good but I would have expected it to be a separate commit with more info.

Reply:
Apologies, I should have explained this more clearly. The change was because I was testing accessibility with the keyboard and noticed it was not working as expected. After reviewing our `dpl-react` implementation, I found that it was set up correctly there. As a result, I decided to add the necessary changes to ensure proper functionality.